.gitignore

```diff
@@ -11,3 +11,4 @@ debug.log
 .git.cache
 *.xlsx
 tests/__xlsx__/xlsx_demo_data/xl/printerSettings
+tools/owl_templates/_compiled/owl_compiled_templates.cjs
```

Comment:
this needs to be backported to all version. Otherwise when we'll checkout a previous branch, the file will be there and git won't ignore it.
An alternative is to locate the compiled template file to the `build` folder

Reply:
I was planning to backport it afterward. And also take the occasion to backport the .gitignore changes of the .xlsx.

But I can put the generated file in build/ if you want.